mmselfsup/models/algorithms/mae_linprobe.py

```diff
@@ -0,0 +1,69 @@
+# Copyright (c) OpenMMLab. All rights reserved.
```

Comment:
'mae_linprobe.py' and 'mae_finetune.py' can be merged, two files are kind of redundant.

Reply:
Or modifying the original classification.py to avoid the code redundancy.